src/tokenizer.rs

```diff
@@ -448,10 +470,25 @@ impl fmt::Debug for Location {
 }
 
 impl Location {
-    pub fn of(line: u64, column: u64) -> Self {
+    /// Return an "empty" / unknown location
+    pub fn empty() -> Self {
+        Self { line: 0, column: 0 }
+    }
+
+    /// Create a new `Location` for a given line and column
+    pub fn new(line: u64, column: u64) -> Self {
         Self { line, column }
     }
 
+    /// Create a new location for a given line and column
+    ///
+    /// Alias for [`Self::new`]
+    // TODO: remove / deprecate in favor of` `new` for consistency?
```

Comment:
I found the `Location::of` API kind of confusing -- I would expect that creating a new Rust object would be done via `::new()` so I added a `new()`

I am thinking perhaps we can deprecate `Location::of` maybe to help downstream users

Reply:
That sounds reasonable to me!